app.rb

```diff
@@ -59,13 +59,14 @@ def soft_redirect(url, page_title)
 end
 
 get '/:country/' do |country|
-  @country = ALL_COUNTRIES.find { |c| c[:url] == country } || pass
-  @page_title = "EveryPolitician: #{@country[:name]}"
+  @page = Page::Country.new(country, ALL_COUNTRIES)
```

Comment:
I don't think you want to be passing things like ALL_COUNTRIES around. You should be able to just call `EveryPolitician.country(country)` in the Page class instead.